cumulus_library_opioid/vocab/common/expand_rules/expand_rules.sql

```diff
@@ -23,6 +23,8 @@ alter table expand_rules add column include boolean default NULL;
 update expand_rules set include = TRUE  where lower(rule) = 'yes';
 update expand_rules set include = FALSE where lower(rule) = 'no';
 
+--source of UMLS_TTY needs to be provided
```

Comment:
Is that a TODO or an explanation?

Reply:
this is a note for when i review state with @comorbidity